src/style.css

```diff
@@ -1,3 +1,82 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  src: url('https://fonts.googleapis.com/css?family=Roboto:500&display=swap');
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+.navigation__wraper {
```

Comment:
There's a typo in the class name `navigation__wraper`. It should be `navigation__wrapper` to match the HTML and ensure the styles are applied correctly.